Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -133,14 +138,20 @@ echo disk > /sys/power/state
 		Write-LogInfo "Rebooting All VMs!"
 		$TestProvider.RestartAllDeployments($AllVMData)
 
-		# Check the VM status before hibernation
-		$vmStatus = Get-AzVM -Name $vmName -ResourceGroupName $rgName -Status
-		if ($vmStatus.Statuses[1].DisplayStatus -eq "VM running") {
-			Write-LogInfo "$($vmStatus.Statuses[1].DisplayStatus): Verified successfully VM status is running before hibernation"
-		} else {
-			Write-LogErr "$($vmStatus.Statuses[1].DisplayStatus): Could not find the VM status before hibernation"
-			throw "Can not identify VM status before hibernate"
-		}
+		For ($iteration=1;$iteration -le $defaultHibernateLoop; $iteration++) {
```

Comment:
for ($iteration=1; $iteration -le $defaultHibernateLoop; $iteration++)

Reply:
Fixed in 7d010730f335277aad9faf45c4f4b50adf182986